ElementX/Sources/FlowCoordinators/RoomFlowCoordinator.swift

```diff
@@ -311,7 +311,8 @@ class RoomFlowCoordinator: FlowCoordinatorProtocol {
         
         let timelineItemFactory = RoomTimelineItemFactory(userID: userID,
                                                           mediaProvider: userSession.mediaProvider,
-                                                          attributedStringBuilder: AttributedStringBuilder(permalinkBaseURL: appSettings.permalinkBaseURL),
+                                                          attributedStringBuilder: AttributedStringBuilder(permalinkBaseURL: appSettings.permalinkBaseURL,
+                                                                                                           mentionBuilder: MentionBuilder(mentionsEnabled: ServiceLocator.shared.settings.mentionsEnabled)),
```

Comment:
Can use the injected settings for this.
```suggestion
                                                                                                           mentionBuilder: MentionBuilder(mentionsEnabled: appSettings.mentionsEnabled)),
```